prov/gni/src/gnix_fabric.c

```diff
@@ -253,6 +253,7 @@ static struct fi_info *_gnix_allocinfo(void)
 	gnix_info->domain_attr->mr_iov_limit = 1;
 	gnix_info->domain_attr->caps = GNIX_DOM_CAPS;
 	gnix_info->domain_attr->mode = 0;
+	gnix_info->domain_attr->mr_cnt = gnix_default_user_registration_limit;
```

Comment:
Please set the value to (65536) for now. Basic registration (which is the current default for the GNI provider) has a much larger pool of registrations than the default specified in that variable (which was/is intended for use with scalable memory registration.